extensions/amp-story/1.0/amp-story.css

```diff
@@ -1001,4 +1001,28 @@ amp-story-quiz {
   /* 224px = the width of a narrow phone (320px) - tap-safe margins (48px * 2) */
   min-width: 224px !important;
   max-width: 400px !important;
+  --accent-color: #005AF0;
```

Comment:
All these CSS variables are very broad and should be prefixed with something (`quiz` or `reaction` or..)

Reply:
I vote `quiz` 😃

Reply:
What would it look like if we iterate to ship polls tomorrow then? Wouldn't we want a unified theming, so whoever implements quizzes also has polls support for free?

Reply:
Alternatively, the publisher won't necessarily want the same configuration for polls and quizzes, it might be strange to force one value that covers both.

A compromise I could strike is prefixing the more general CSS custom properties `--reaction-`, and having `--quiz-` and `--poll-` custom properties inherit from that, but have their own values in case the publisher wanted to override.

Reply:
> Alternatively, the publisher won't necessarily want the same configuration for polls and quizzes, it might be strange to force one value that covers both.

From this PR, publishers will have to either set a global CSS variable that will apply to all quizzes, or target a specific quiz with its ID. So they can still target polls or quizzes with either their specific ID, or `amp-story-quiz {/* variables*/} amp-story-poll {/* variables */}`

I know I'm very annoying but I really think launching quizzes + polls + sliders at the same time is the way to go and would like the next iterations to be as easy and fast as possible, as it unlocks a lot of new business use cases for publishers to receive feedback.

Reply:
Ah, true!  Common naming could make a lot of sense here